button_shutdown_1         : 8
button_shutdown_2         : 9

button_move_enable        : 4
axis_vel_surge            : 1
axis_vel_sway             : 0
axis_vel_angular          : 3 

button_kick_enable        : 5
button_kick_straight      : 3
button_kick_chip          : 2
axis_kick_power           : 7

button_dribble_enable     : 7
axis_dribble_power        : 7

button_id_enable          : 9
axis_id_change            : 7

button_color_enable       : 8
axis_color_change         : 7

button_all_id_1           : 4
button_all_id_2           : 5
button_all_id_3           : 7
button_all_id_4           : 9

# for indirect control
button_path_enable        : 5
button_add_pose           : 1
button_delete_path        : 0
button_send_target        : 3
